apps/webpack.config.js

```diff
@@ -80,6 +80,7 @@ const circularDependenciesSet = new Set(circularDependencies);
 // as we see our known circular dependencies, we're gonna remove them from our list. That way,
 // we can report at the end if any circular dependencies have been cleaned up.
 let seenCircles = new Set();
+let numUnresolvedCircles = 0;
```

Comment:
Adding some extra context to output to make debugging easier.

Reply:
Would you mind extracting this change into its own PR? I like the new functionality, but I do want to keep it separate from the cleanup work that's supposed to have zero side effects. I'll also be interested to hear what @thomasoniii thinks about it!